akka-projection-slick/src/test/scala/akka/projection/slick/SlickSpec.scala

```diff
@@ -4,33 +4,8 @@
 
 package akka.projection.slick
 
-import scala.concurrent.Await
-import scala.concurrent.duration._
-
 import akka.actor.testkit.typed.scaladsl.LogCapturing
 import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
-import akka.projection.slick.internal.SlickOffsetStore
-import akka.projection.slick.internal.SlickSettings
-import akka.projection.testkit.scaladsl.ProjectionTestKit
 import com.typesafe.config.Config
-import slick.basic.DatabaseConfig
-import slick.jdbc.H2Profile
-
-abstract class SlickSpec(config: Config) extends ScalaTestWithActorTestKit(config) with LogCapturing {
-
-  val dbConfig: DatabaseConfig[H2Profile] = DatabaseConfig.forConfig(SlickSettings.configPath, config)
-
-  val offsetStore = new SlickOffsetStore(dbConfig.db, dbConfig.profile, SlickSettings(system))
-
-  val projectionTestKit = new ProjectionTestKit(testKit)
-
-  override protected def beforeAll(): Unit = {
-    // create offset table
-    Await.ready(offsetStore.createIfNotExists, 3.seconds)
-  }
 
-  override protected def afterAll(): Unit = {
-    super.afterAll()
-    dbConfig.db.close()
-  }
-}
+abstract class SlickSpec(config: Config) extends ScalaTestWithActorTestKit(config) with LogCapturing
```

Comment:
Then I don't think it's much value in having this at all. Good thing of not creating internal test abstractions is that if users look at our tests they can use them as inspiration for their own tests.

That's the approach we have taken with most Akka Typed tests, while in classic tests we have AkkaSpec, and users are confused when we point them to the tests.

Reply:
ok, I will remove that trait. 

I did the same on #192, but will send another PR on top of #210 to avoid going back and forth and unnecessary rebases.